pkg/labels/labels_test.go

```diff
@@ -179,7 +207,7 @@ func (s *LabelsSuite) TestLabelSliceSHA256Sum(c *C) {
 	}
 	sha256sum, err := LabelSliceSHA256Sum(labels)
 	c.Assert(err, IsNil)
-	c.Assert(sha256sum, DeepEquals, "180d22655dec0e78f62c3e4ac17f5994baedbdfc5c882cbbd668e2628c44f320")
+	c.Assert(sha256sum, DeepEquals, "330d57790ccf8cd0b10000d6297baa5fa27b9d6e1cb228055b45dc6e440ea4da")
```

Comment:
Why do you think this is a problem? Are you worried about users running 0.8 and 0.9 in parallel?